app/src/main/res/values/strings.xml

```diff
@@ -126,6 +126,7 @@
     <string name="confirmButtonAdCreation">Post</string>
     <string name="toolbarTitle_AdCreation">Create an ad</string>
     <string name="snackbarFailed_AdCreation">There was an error while posting your ad, please try again later</string>
+    <string name="snackbarFailedLocation_AdCreation">There was an error while posting your ad, please check that you put the right location!</string>
```

Comment:
Maybe in this case since it's a forced requirement and not an error, I would drop the `There was an error while posting your ad` and keep only the second part

Reply:
Yes maybe is better!